packages/aws-cdk/lib/assets.ts

```diff
@@ -10,7 +10,7 @@ import { zipDirectory } from './archive';
 import { prepareContainerAsset } from './docker';
 import { debug, success } from './logging';
 
-export async function prepareAssets(stack: SynthesizedStack, toolkitInfo?: ToolkitInfo, ci?: boolean): Promise<CloudFormation.Parameter[]> {
+export async function prepareAssets(stack: SynthesizedStack, ci: boolean, toolkitInfo?: ToolkitInfo): Promise<CloudFormation.Parameter[]> {
```

Comment:
Had to reorder `ci` and `toolkitInfo` here (_a required parameter cannot follow an optional parameter_)